src/components/Header/index.js

```diff
@@ -1,18 +1,54 @@
 const React = require('react');
+const T = require('prop-types');
 const NavLink = require('react-router-dom').NavLink;
 const Classes = require('./styles.scss');
 
-module.exports = () => (
-
-    <div>
-        <h1>Strangeluv</h1>
-        <NavLink exact to='/' activeClassName={Classes.activeRoute}>
-            Home
-        </NavLink>
-        {' · '}
-        <NavLink to='/counter' activeClassName={Classes.activeRoute}>
-            Counter
-        </NavLink>
-    </div>
-
-);
+module.exports = class Header extends React.Component {
+
+    static propTypes = {
+        logout: T.func.isRequired,
+        isAuthenticated: T.bool.isRequired
+    };
+
+    renderNotAuthenticated = () => {
+
+        return (
+
+            <React.Fragment>
+                {' · '}
+                <NavLink to='/sign-up' activeClassName={Classes.activeRoute}>Sign Up</NavLink>
+                {' · '}
+                <NavLink to='/login' activeClassName={Classes.activeRoute}>Login</NavLink>
+            </React.Fragment>
+        );
+    }
```

Comment:
You can also just,
```js
    return <React.Fragment>
        {stuff}
    </React.Fragment>;
```

But if you like the extra parens, there's nothing wrong with that either!  Just expect to see both styles kickin around.

Reply:
I kinda like the extra parens (with the extra newlines I like to toss in 😛)